### ISSUE TITLE
error when using Blender 3.2

### ISSUE THREAD

**Issue by hydralica** [Aodaruma/BPMmarker#2]
```
Python: Traceback (most recent call last):
  File "\BPMmarker.py", line 89, in execute
TypeError: TimelineMarkers.new(): error with keyword argument "frame" -  Function.frame expected an int type, not float
```

**Comment by hydralica** [comment]
I found a fix. change `frame=frame+start` on line 90 to `frame=int(frame+start)`

**Comment by Aodaruma** [comment]
thank you for sending PR! I merged #3
